target/linux/ar71xx/config-4.4

```diff
@@ -47,6 +47,7 @@ CONFIG_ATH79_MACH_AP136=y
 CONFIG_ATH79_MACH_AP143=y
 CONFIG_ATH79_MACH_AP147=y
 CONFIG_ATH79_MACH_AP152=y
+CONFIG_ATH79_MACH_AP531B0=y
```

Comment:
Hi @roboidler,

> i double check that in my env, original attributes of config-4.4 is 664, not 644. is there something wrong?

As far as I remember, git tracks only executable bit, so you shouldn't care about other attributes.